OpenShift/Makefile

```diff
@@ -1,14 +1,19 @@
 .PHONY: default all requirements configure
 default: requirements configure
 
-all: default
+all: cluster post_install
```

Comment:
I am thinking the default should be just create,   @e-minguez suggested leaving Post install as a manual step when I suggested his changes

@e-minguez  - opinion of making post_install part of all - I am okay with that

Reply:
I'm ok

Reply:
From what I gathered on the CNV-2.1 PR, the post_install steps will be necessary to deploy CNV-2.1, which is part of the appliance so I figured it should be part of all. However, if you guys think it is better to keep it out of all I can make that change np